tests/Greenspun/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+
+if (NOT MSVC)
+    add_executable(greenspun_repl REPL.cpp)
```

Comment:
Any ideas where to put it?

Reply:
Hm. Many other CLI tools seem to be in `arangosh/`, so maybe `arangosh/Greenspun/greenspun.cpp` or so? I'm not quite sure. @jsteemann, WDYT?

Reply:
Will this be a separate tool that we ship with ArangoDB? Then it should go into arangosh.
If it is just for our internal debugging and not expected to be used by end users/customers, it can be anywhere

Reply:
It was never intended to be shipped or used by a customer.